packages/rolldown/tests/cli/cli-e2e.test.ts

```diff
@@ -0,0 +1,45 @@
+import { describe, test, it, expect } from 'vitest'
+import { execSync } from 'node:child_process'
+import { $ } from 'execa'
+
+import { projectDir, testsDir } from '@tests/utils'
+
+function cliFixturesDir(...joined: string[]) {
+  return testsDir('cli/fixtures', ...joined)
+}
+
+describe('should not hang after running', () => {
+  test.skip('basic', async () => {
+    const cwd = cliFixturesDir('no-config')
+    const _ret = execSync('rolldown', { cwd })
+  })
+})
+
+describe('args', () => {
+  it('should render help message for empty args', async () => {
+    const stdout = execSync('rolldown', {
+      cwd: projectDir(),
+      encoding: 'utf-8',
+    })
+    // FIXME: `citty` emit nothing
+    expect(stdout).toMatchSnapshot(`""`)
```

Comment:
@pi0 I guess this might be related to `consola`, but I'm not sure why. I looked the code, `consola` also write texts to `process.stdout`. Not sure why can't capture them.